### PR TITLE
Translations update from Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-hts/) for Kodi add-ons: PVR clients/pvr.hts.

Current translation status:

Weblate translation status

### CODE REVIEW

File: pvr.hts/resources/language/resource.language.de_de/strings.po [hunk 1]
```diff
@@ -60,6 +60,14 @@ msgctxt "#30010"
 msgid "Timer settings"
 msgstr "Timer"
 
+msgctxt "#30011"
+msgid "Use HTTPS"
+msgstr "Use HTTPS"
```
Comment (ksooo) [inline pvr.hts/resources/language/resource.language.de_de/strings.po]: This is definitely no German translation. Where does this string come from?

**Comment by ksooo** [comment]
do not merge.

File: pvr.hts/resources/language/resource.language.de_de/strings.po (hunk 1 above)
Comment (ksooo) [reply pvr.hts/resources/language/resource.language.de_de/strings.po]: Reason is that the en_GB strings.po is not correct. We need to fix that.

Comment (gade01) [reply pvr.hts/resources/language/resource.language.de_de/strings.po]: Yeah, it needs to be blank here: https://github.com/kodi-pvr/pvr.hts/blob/Matrix/pvr.hts/resources/language/resource.language.en_gb/strings.po#L67

**Comment by Jalle19** [comment]
@ksooo should this still not be merged?

**Review by gade01 — APPROVED** [review]
Everything looks very good to me.

- Some new translations (no replacements)
- Some translation fixes done by me
- New strings

**Review by phunkyfish — APPROVED** [review]

**Comment by phunkyfish** [comment]
Ok, all the automation is in now @ksooo. When you get around to merging (no rush on this)  this another PR will be created for the release with the changelog entry. Then simply merge and release that.

It has worked consistently and well across all pvr addons so we could also automate the changelog and release step. In that case only this single PR would need to be merged.

**Pull request merged**